src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1115,6 +1116,13 @@ public function testCanCheckIfTerminalIsInteractive()
         $inputStream = $tester->getInput()->getStream();
         $this->assertEquals($tester->getInput()->isInteractive(), @posix_isatty($inputStream));
     }
+
+    public function testAddApplicationReturnsNullWhenCommandIsNotEnabled()
+    {
+        $application = new CustomDefaultCommandApplication();
+
+        $this->assertNull($application->add(new \DisabledCommand()));
```

Comment:
Here $application is always new object so it will null and you assert it should be null. I think it is not helpful testcase.